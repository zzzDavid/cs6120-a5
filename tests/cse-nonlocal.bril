# CMD: bril2json < {filename} | python ../main.py -cse | bril2txt
@main {
  x: int = const 4;
  sum1: int = add x x;
  jmp .label;
.label:
  sum2: int = add x x;
  print sum2;
}
