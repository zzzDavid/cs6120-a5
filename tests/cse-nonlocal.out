@main {
.start:
  x: int = const 4;
  sum1: int = add x x;
  jmp .label;
.label:
  sum2: int = id sum1;
  print sum1;
}
